### ISSUE TITLE
editing datagrid

### ISSUE THREAD

**Issue by MannusEtten** [MyToolkit/MyToolkit#43]
hello,

i like the toolkit and especially the datagrid, does it also have editing and deleting features?

**Comment by rsuter** [comment]
You can implement custom editing yourself with the templated culumn class...

**Comment by rsuter** [comment]
Use the DataGridTemplatedColumn class for this

**Comment by MannusEtten** [comment]
okay thanks for the information, i decided that datagrids are not really recommended anymore for windows 10 app-development.

it is not described in the UI guidelines anymore, they recommend master/detail-views as the replacement for this.

**Comment by rsuter** [comment]
This is true for "simple" data consumption apps. However UWP apps will eventually replace desktop (WPF) applications and some of them need a way to display/edit large data sets without switching the view for each entry...
